package.json

```diff
@@ -1,14 +1,19 @@
 {
   "name": "@corbado/node-sdk",
-  "version": "1.1.16",
+  "version": "2.0.0",
   "description": "This Node.js SDK eases the integration of Corbado's passkey-first authentication solution.",
-  "main": "dist/cjs/index.js",
-  "module": "src/index.js",
+  "module": "src/sdk.ts",
   "type": "module",
+  "main": "lib/sdk.js",
+  "files": [
+    "lib/"
+  ],
   "scripts": {
-    "build-es5": "babel src -d dist/cjs --presets=@babel/preset-env --plugins=@babel/plugin-transform-modules-commonjs",
-    "build": "rm -fr dist/* && npm run build-es5 && ./fixup",
-    "test": "mocha --reporter spec"
+    "generate-client": "openapi-generator-cli generate -g typescript-axios -i ../corbado/backend/openapi/backend_api_public.yml -o src/sdk/entity/api",
```

Comment:
1. Rename to "generate-openapi"
2. Download spec file (see PHP)
3. Use openapi-generator docker container (see PHP)